index.html

```diff
@@ -52,4 +52,184 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
+          </div>
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="img-container">
+        <img class="work-image" src="./Images/Tonic-two.png" alt="work-Image" />
+      </div>
+      <div class="card">
+        <div class="card-element">
+          <h2 class="card-title">Tonic</h2>
+          <div class="card-section">
+            <p class="para-one">CANOPY</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">Back End Dev</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">2015</p>
+          </div>
+          <p class="card-body">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tech-stack">
+            <button class="tech-items" type="button">html</button>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
+          </div>
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="img-container multipost">
+        <img class="work-image" src="./Images/Multiple-two.png" alt="work-Image" />
+      </div>
+      <div class="card multipost">
+
+        <div class="card-element">
+          <h2 class="card-title">
+            Multi-Post Stories
+          </h2>
+          <div class="card-section">
+            <p class="para-one">CANOPY</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">Back End Dev</p>
+            <i class="fa fa-circle"></i>
+            <p class="para-two">2015</p>
+          </div>
+          <p class="card-body">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tech-stack">
+            <button class="tech-items" type="button">html</button>
+            <button class="tech-items" type="button">css</button>
+            <button class="tech-items" type="button">javascript</button>
+          </div>
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </div>
+
+    </div>
+    <div class="about-sec">
+      <div class="about-content-container">
+        <h2 class="about-intro">About<br />
+          Myself
+        </h2>
+        <p class="about-description">
+          Hello I’m a software developer! I can help you build a product,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <h3 class="bottom-connect about-connect">LET’S CONNECT</h3>
+        <ul class="icon-container about-icon">
```

Comment:
- [ ] Please, use an `a` tag for this, since it will be used for the purpose that it describes

Reply:
Thanks for the suggestion. I have done the changes.

Reply:
This issue is resolved 👍